dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -41,6 +41,16 @@ properties:
       description: |
         Operation mode of the PHY interface.
 
+    peripheral-id:
+      type: int
+      default: 0
```

Comment:
This default is new and its value must be justified to follow the rules in the DT bindings documentation. This is a blocking change request.